ExodusMovement/lerna-release-action
# feat: close previous PRs for the same packages

This PR improves the version process to close existing release PRs that release the exact same set of packages as the new PR to be created. It also leaves a comment such as `Closing in favor of <pr>`.

Closes #25 

## Testplan

1. Release batcave,wayne-tower in the test repo
2. Release batcave in the test repo
3. Verify the PR from 1 is not closed
4. Release batcave again
5. Verify the PR from 2 is closed
6. Releae batcave,wayne-tower
7. Verify the PR from 1 is closed

Action is added in the test repo. Example of closed PRs:
https://github.com/ExodusMovement/lerna-version-selectively/pull/363
https://github.com/ExodusMovement/lerna-version-selectively/pull/362
